Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 LiteFarm.org
+# This file is part of LiteFarm.
+#
+# LiteFarm is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# LiteFarm is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+
+include .env.secrets
+
+secrets:
+	@if [ ! -f .env.secrets ]; then echo "Please copy .env.secrets.default to .env.secrets and fill the variables"; exit 1; fi;
+	@if grep -Fq "=?" .env.secrets ; then echo "Please fill all the secrets in .env.secrets"; exit 1; fi;
+
+db_dropped:
+	@docker compose down
+	@docker volume prune -a -f
+
+api_env_file: ./packages/api/.env.default secrets
+	@cp ./packages/api/.env.default ./packages/api/.env
+	@sed -i 's/GOOGLE_API_KEY=?/GOOGLE_API_KEY=$(GOOGLE_API_KEY)/g' ./packages/api/.env
```

Comment:
With default MacOS `sed` (I had to look this up when I got an error) the syntax needs to be `@sed -i ''`, so I think a conditional to check the platform and format the sed command is needed. (A mac user could also `brew install gnu-sed` and update the PATH but for onboarding I don't think that should be required!)